### PR TITLE
Added jquery.dotdotdot and jquery.jScrollPane

### PR DESCRIPTION
**Pull request by gig177**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -233,7 +233,9 @@
   "jquery.cookie": "github:carhartl/jquery-cookie",
   "jquery.hammer": "github:hammerjs/jquery.hammer.js",
   "jquery.tipTip": "github:drewwilson/TipTip",
+  "jquery.dotdotdot": "github:BeSite/jQuery.dotdotdot",
```
Comment (guybedford) [inline registry.json]: This should be further up to be alphabetical.

**Comment by guybedford** [comment]
This looks great, thanks.

File: registry.json (hunk 1 above)
Comment (gig177) [inline registry.json]: Ok. I did it

File: package-overrides/github/vitch/jScrollPane@2.0.22.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "main": "script/jquery.jscrollpane",
+    "dependencies": {
+        "jquery": "*"
```
Comment (guybedford) [inline package-overrides/github/vitch/jScrollPane@2.0.22.json]: `css` should be a dependency here as well.

**Comment by guybedford** [comment]
Thanks, just saw the other point now too

File: package-overrides/github/vitch/jScrollPane@2.0.22.json (hunk 2 above)
Comment (gig177) [inline package-overrides/github/vitch/jScrollPane@2.0.22.json]: Should i specify the clean-css as dependency too?

Comment (guybedford) [inline package-overrides/github/vitch/jScrollPane@2.0.22.json]: No just `css` is fine.

**Comment by gig177** [comment]
Is it all right?

**Pull request merged**